### PR TITLE
Add universal "New" button

### PR DESCRIPTION
**Pull request by mistercrunch**
As discussed here https://github.com/apache/incubator-superset/issues/6668

screen shot 2019-01-12 at 11 04 06 am
screen shot 2019-01-12 at 11 03 54 am

screen shot 2019-01-12 at 10 13 28 am

### CODE REVIEW

File: superset/views/core.py [hunk 1]
```diff
@@ -2067,6 +2067,18 @@ def warm_up_cache(self):
             [{'slice_id': slc.id, 'slice_name': slc.slice_name}
              for slc in slices]))
 
+    @has_access_api
+    @expose('/dashboard/new/')
+    def new_dashboard(self):
```
Comment (hughhhh) [inline superset/views/core.py]: Can we get a small unit test here

**Comment by mistercrunch** [comment]
@hughhhh I added the unit test

File: superset/views/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# pylint: disable=C,R,W
```
Comment (bolkedebruin) [inline superset/views/dashboard.py]: Please add ASF license header

**Review by bolkedebruin — CHANGES_REQUESTED** [review]

**Review by hughhhh — APPROVED** [review]
Alt Text

**Comment by hughhhh** [comment]
@mistercrunch LGTM outside of @bolkedebruin request for ASF

**Comment by dorq** [comment]
@mistercrunch As discussed [here](https://github.com/apache/incubator-superset/issues/6668#issuecomment-454218808) can you confirm the text for the filter is "Filter List"?

**Comment by dorq** [comment]
...and will this PR also address the removal of the "+" button in the CRUD UI?

**Comment by dorq** [comment]
@mistercrunch The issue suggested "Filter List", like this: 
image

**Pull request merged**